### DISCUSSION TITLE
Is the OPi5Plus NVMe WiFi card working in the OS?

### DISCUSSION THREAD

**Discussion by traxap** [Joshua-Riek/ubuntu-rockchip#497] [Q&A] [answered]
I purchased the Orange Pi 5 Plus WiFi NVMe card but it does not detect any WiFi networks even though I have wireless network and Personal Hotspot active in close proximity. 
Do I need to activate wireless options in the software?  If software settings are required, where do I find them and what settings should I select? 
Any help would be appreciated.
Thank you.

**Comment by bagong** [comment]
There are quite a few models, and you must make sure you have one of the latest releases of Joshua's kernel. I have an Intel card and it works fine now (don't know which one it is exactly, too lazy to look it up). I had a LCL RTL8852BE before that - that actually came with the board (bought through aliexpress, not directly from opi/Xulong) - and that used not to work (both wifi and bluetooth not). It might however work now, something changed in the drivers a couple of months ago. The wifi should activate automatically, so you just need to log into the network - if it is supported. if it doesn't work, you could in theory try to find drivers for your card, but that's neither trivial, nore likely to exist, as the hardware producers try to build network-boards that are clones of existing supported models rather than provide their own drivers.

**Comment by traxap** [comment]
Hi bagong,

Thank you for the quick reply.  I have the Realtek RTL8852BE card you mentioned in your post.  I will return it and get the intel one. I would appreciate it if you can tell me the model number of your card.

**Comment by bagong** [comment]
Ah, found it in my Amazon orders (mid October):

Intel OSGEAR-NEW Dual Band Wireless-AX200NGW WLA/Wi-Fi 6 AX200 2230 2x2 AX+ Bluetooth 5.0, M.2/A-E-Key (AX200.NGWG) Wi-Fi 6 AX200 with vPro, 2.4GHz/5GHz WLAN, Bluetooth 5.0, M.2/A-E-Key 80 2.11ax

**Reply by traxap** [reply]
Thank you bagong.  I will post when I get the card and install it.

**Comment by Joshua-Riek** [comment, marked as answer]
This WiFi card should be working perfectly with the latest OS release. Look in the system WiFi settings and make sure the antennas are attached.

**Reply by bagong** [reply]
You mean the Realtek or the Intel?

**Reply by traxap** [reply]
Thank you Josh and bagong,  

Following your advice I finally got my Realtek RTL8852BE... card to work. 

Steps:. 
I set up wireless in settings.  I disconnected and reconnected the card and the antennas, then I rebooted.  After the reboot the WiFi symbol showed up in the taskbar and I was able to see available networks.  With wireless available I was able to set up and connect to WiFi networks including a personal Hotspot. 

Thank you again for your suggestions and support.

P.S. I am keeping the card and plan to cancel my return.
